src/components/date-input/README.md

```diff
@@ -507,6 +529,14 @@ Guidelines can be found at [on our Github repository.](https://github.com/alphag
 
 ## Acknowledgements/credits
 
+*   GDS developers
```

Comment:
There is a card/task in the backlog to remove Acknowledgements/credits section (with which I agree), but I don't understand why they appear/disappear from one update to another 🤔

Reply:
I think the usage of the list component in the README template was broken (it predated passing objects rather than kwargs) – when Joe removed the list component he updated the template to just use a `<ul>` and so now updates to READMEs are bringing them back.

Reply:
Yep, that would make sense.